markets/perps-market/contracts/storage/AsyncOrder.sol

```diff
@@ -168,11 +178,9 @@ library AsyncOrder {
         }
 
         // TODO: validate position size
```

Comment:
I this comment still valid?

Reply:
yes.  there's a `validatePositionSize` function but i haven't looked at it closely; its from v2, but my assumption is we would check the new market size against the maxMarketSize to see if position is valid.